src/components/DataTable.tsx

```diff
@@ -126,7 +127,7 @@ const DataTable = <T extends RowData>({ data, columns, stickyHeader, sorting, on
   };
 
   return (
-    <Table className="text-base">
+    <Table className={`${largerText ? 'govuk-table' : 'text-base'}`}>
```

Comment:
use classes from govuk for this go govuk-body

Reply:
fixed